crates/bevy_pbr/src/light.rs

```diff
@@ -497,8 +497,8 @@ fn ndc_position_to_cluster(
         .clamp(UVec3::ZERO, cluster_dimensions - UVec3::ONE)
 }
 
-const VEC2_HALF: Vec2 = const_vec2!([0.5, 0.5]);
-const VEC2_HALF_NEGATIVE_Y: Vec2 = const_vec2!([0.5, -0.5]);
+const VEC2_HALF: Vec2 = Vec2::from_array([0.5, 0.5]);
```

Comment:
For stuff like this you could change it to `Vec2::new(0.5, 0.5)` or even `Vec2::splat(0.5)`, the array version isn't really necessary unless you prefer it.

Reply:
Thanks for the feedback. I've changed things.